docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -9,24 +9,65 @@ endif::[]
+==== Control rolling restarts during the upgrade
+
+Upgrading the operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods by Kubernetes to apply those changes. If you have a very large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart might be disruptive or inconvenient. To have more control over when the pods belonging to a particular deployment should be restarted, you can <<{p}-exclude-resource,add an annotation>> to the corresponding Elasticsearch, Kibana, or APM Server resources to temporarily exclude them from being managed by the operator. When the time is convenient, you can remove the annotation and let the rolling restart go through.
+
+CAUTION: Once a resource is excluded from being managed by ECK, you will not be able to add/remove nodes, upgrade Stack version, or perform other <<{p}-orchestrating-elastic-stack-applications, orchestration tasks>> by updating the resource manifest. You must remember to remove the exclusion to ensure that your Elastic Stack deployment is continually monitored and managed by the operator.
+
+[source,shell,subs="attributes,callouts"]
+.Exclude Elastic resources from being managed by the operator
+----
+ANNOTATION='common.k8s.elastic.co/pause=true' <1>
+
+# Exclude a single Elasticsearch resource named "quickstart"
+kubectl annotate --overwrite elasticsearch quickstart $ANNOTATION
+
+# Exclude all resources in the current namespace
+kubectl annotate --overwrite elastic --all $ANNOTATION
+
+# Exclude all resources in all of the namespaces:
+for NS in $(kubectl get ns -o=custom-columns='NAME:.metadata.name' --no-headers); do kubectl annotate --overwrite elastic --all $ANNOTATION -n $NS; done
+----
 
-Upgrading from ECK 1.0.0-beta1 to 1.0.0 has some special considerations for users running either:
-- Kubernetes versions < 1.13
-- OpenShift 3.11
+<1> Since ECK 1.1.0, this annotation has been superseded by `eck.k8s.elastic.co/managed=false`
 
-In this case, ECK must be uninstalled and reinstalled (due to link:https://github.com/kubernetes/kubernetes/issues/73752[an upstream bug] that was not backported). To uninstall ECK, see <<{p}-uninstalling-eck>>. Then you can follow the <<{p}-quickstart>> to install anew. You can save your existing `v1beta1` manifests, update them to the `v1` API version, <<{p}-snapshots,take snapshots>> and restore them as well.
+Once the operator has been upgraded and you are ready to let the resource become managed again (triggering a rolling restart of pods in the process), remove the annotation.
+
+
+[source,shell,subs="attributes,callouts"]
+.Resume Elastic resource managment by the operator
+----
+RM_ANNOTATION='common.k8s.elastic.co/pause-' <1>
+
+# Resume management of a single Elasticsearch cluster named "quickstart"
+kubectl annotate elasticsearch quickstart $RM_ANNOTATION
+----
+
+<1> Since ECK 1.1.0, this annotation has been superseded by `eck.k8s.elastic.co/managed-`
```

Comment:
The final dash seems unexpected.

Reply:
It is the way to remove an annotation though. Any suggestions to make it look like it's not a typo?

Reply:
In this case, it is legitimate and I think it's okay.

I'm not sure it will be more clear, we could keep the dash in the command like this:
```kubectl annotate elasticsearch quickstart ${RM_ANNOTATION}-```.

I'm fine with keeping it like this.